client/dist/index.html

```diff
@@ -2,7 +2,6 @@
 <html>
   <head>
     <title></title>
-    <link rel="stylesheet" href="attraction-overview.css"></link>
```

Comment:
It's good to have this here, but I think this would mean you are maintaining two copies of your css file in your project. 

Maybe now that the bundle is handling the css import we can remove this line?